Makefile

```diff
@@ -33,5 +33,6 @@ docs:
 clean:
 	$(MAKE) -C docs clean
 
-requirements.txt: setup.py
-	pip-compile setup.py
+requirements.txt: setup.py requirements.Dockerfile
```

Comment:
Could be one-liner without any extra files...

```
docker run --rm -u $(id -u) -v $PWD:/tmp/app -w /tmp/app registry.access.redhat.com/ubi8/python-39 bash -c 'pip install pip-tools && pip-compile'
```

Reply:
Fixed